packages/prisma-client/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@webstudio-is/prisma-client",
+  "version": "0.0.1",
+  "description": "Webstudio Prisma layer",
+  "author": "Webstudio <github@webstudio.is>",
+  "homepage": "https://webstudio.is",
+  "scripts": {
+    "checks": "tsc --noEmit",
+    "build:prisma": "prisma format && prisma generate",
+    "migrate": "dotenv -e ../../apps/designer/.env -- prisma migrate"
```

Comment:
this makes it so prisma will read the env from the designer and anyone just needs to have one env file

Reply:
yeah, the problem is that prisma package ideally is not supposed to know anything about designer from abstraction perspective

why does it need to read the .env, can't it be already read before one runs prisma migrate?

Reply:
The two options are:

- We have two .env files, one here and one in the designer
- We do this

What do you prefer?

Reply:
Here is the prisma page about it: https://www.prisma.io/docs/guides/development-environment/environment-variables/managing-env-files-and-setting-variables